engine/api/src/main/java/io/deephaven/engine/table/Table.java

```diff
@@ -1761,6 +1761,23 @@ Table updateBy(@NotNull final UpdateByControl control,
     Table updateBy(@NotNull final UpdateByControl control,
             @NotNull final Collection<? extends UpdateByClause> operations);
 
+    /**
+     * <p>
+     * Create a table with the same {@link #getRowSet() rowSet} as its parent that will perform the specified set of row
+     * based operations to it. As opposed to {@link #update(String...)} these operations are more restricted but are
+     * capable of processing state between rows.This operation will group the table by the specified set of keys if
+     * provided
```

Comment:
Truncated. I think you mean 
```suggestion
     * provided before applying the operations.
```